src/deploy/NVA_build/Tests.Dockerfile

```diff
@@ -26,9 +27,10 @@ WORKDIR /root/node_modules/noobaa-core/
 #   Size: ~ 83.9 MB
 #
 ##############################################################
-RUN ./src/test/system_tests/ceph_s3_tests/test_ceph_s3_deploy.sh $(pwd)
-RUN cd ./src/test/system_tests/ceph_s3_tests/s3-tests/ && \
-    ./bootstrap
+RUN ./src/test/system_tests/ceph_s3_tests/test_ceph_s3_deploy.sh $(pwd) 
+RUN cd ./src/test/system_tests/ceph_s3_tests/ && \
+    chgrp -R 0 s3-tests && \
+    chmod -R g=u s3-tests
```

Comment:
Would you please add comment about this change?

Reply:
its about adding group permissions to the s3-tests repo. it copies the permissions for the user (currently root). it adds write permissions to group. its so tox can create its files